src/Microsoft.Identity.Client/Internal/ClientAssertionCertificateWrapper.cs

```diff
@@ -1,64 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Globalization;
-using System.Security.Cryptography.X509Certificates;
-using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
-using Microsoft.Identity.Client.Utils;
-
-namespace Microsoft.Identity.Client.Internal
-{
-#if !ANDROID_BUILDTIME && !iOS_BUILDTIME && !WINDOWS_APP_BUILDTIME && !MAC_BUILDTIME
-
-    /// <summary>
-    /// Certificate for a client assertion. This class is used in one of the constructors of <see cref="ClientCredential"/>. ClientCredential
-    /// is itself used in the constructor of <see cref="ConfidentialClientApplication"/> to pass to Azure AD a shared secret (registered in the
-    /// Azure AD application)
-    /// </summary>
-    /// <seealso cref="ClientCredential"/> for the constructor of <seealso cref="ClientCredential"/>
-    /// with a certificate, and <seealso cref="ConfidentialClientApplication"/>
-    /// <remarks>To understand the difference between public client applications and confidential client applications, see https://aka.ms/msal-net-client-applications</remarks>
-    internal sealed class ClientAssertionCertificateWrapper
```

Comment:
This was a wrapper that simply contained the client credential certificate properties with very little functionality. Everything here has been merged into ClientCredentialWrapper.cs to reduce nested wrappers and confusion.